CaseFacadeEjbTest fails on local build

<!--
If you've never submitted an issue to the SORMAS repository before or this is your first time using this template, please read the Contributing guidelines (https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md) for an explanation of the information we need you to provide. You don't have to remove this comment or any other comment from this issue as they will automatically be hidden.
-->
### Bug Description
There are test failures in Maven build on local machine.

### Steps to Reproduce
Maven build on a local machine fails:
```
SORMAS-Project/sormas-base$ mvn clean install

==>

[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   CaseFacadeEjbTest.testCaseCompletenessWhenCaseFound:2044 
Expected: <2021-08-31 14:03:53.138>
     but: was <2021-08-31 14:03:53.15>
[ERROR]   CaseFacadeEjbTest.testDoesEpidNumberExist:1407
[ERROR]   CaseFacadeEjbTest.testGenerateEpidNumber:1158 expected:<COU-REG-DIS-21-001> but was:<null>
[INFO] 
[ERROR] Tests run: 566, Failures: 3, Errors: 0, Skipped: 3
```

By narrowing down the cause, the interfering tests can be identified as `CaseFacadeEjbTest` and `CaseClassificationLogicTest`:
```
SORMAS-Project/sormas-backend$ mvn test -Dtest=CaseClassificationLogicTest,CaseFacadeEjbTest

==>

[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   CaseFacadeEjbTest.testCaseCompletenessWhenCaseFound:2044 
Expected: <2021-08-31 14:23:28.374>
     but: was <2021-08-31 14:23:28.386>
[ERROR]   CaseFacadeEjbTest.testDoesEpidNumberExist:1407
[ERROR]   CaseFacadeEjbTest.testGenerateEpidNumber:1158 expected:<COU-REG-DIS-21-001> but was:<null>
[INFO] 
[ERROR] Tests run: 65, Failures: 3, Errors: 0, Skipped: 0
```

Inspecting `CaseClassificationLogicTest` hints to the setting of property `FEATURE_AUTOMATIC_CASE_CLASSIFICATION` before test.

### Expected Behavior
Test executions should be independent from previously run tests.

### Solution
When resetting Mocks in `MockProducer`, reset properties, too.

### System Details
<!-- Mandatory; you only have to specify the Server URL if the error appeared on a publicly available test server -->
* Device: Linux PC
* SORMAS version: 1.64.0-SNAPSHOT
* Android version/Browser:
* Server URL:
* User Role:

### Additional Information
To be noted: this behaviour is reproducible on my local machine, but does not seem to affect the CI builds.

--- a/sormas-backend/src/test/java/de/symeda/sormas/backend/MockProducer.java
+++ b/sormas-backend/src/test/java/de/symeda/sormas/backend/MockProducer.java
@@ -39,8 +39,8 @@ import javax.transaction.UserTransaction;
 
 import de.symeda.sormas.api.utils.InfoProvider;
 import de.symeda.sormas.backend.common.ConfigFacadeEjb;
-import de.symeda.sormas.backend.sormastosormas.crypto.SormasToSormasEncryptionFacadeEjb.SormasToSormasEncryptionFacadeEjbLocal;
 import de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService;
+import de.symeda.sormas.backend.sormastosormas.crypto.SormasToSormasEncryptionFacadeEjb.SormasToSormasEncryptionFacadeEjbLocal;
 import de.symeda.sormas.backend.sormastosormas.rest.SormasToSormasRestClient;
 import de.symeda.sormas.backend.sormastosormas.rest.SormasToSormasRestClientProducer;
 
@@ -66,9 +66,7 @@ public class MockProducer {
 	private static Session mailSession;
 
 	static {
-		properties.setProperty(ConfigFacadeEjb.COUNTRY_NAME, "nigeria");
-		properties.setProperty(ConfigFacadeEjb.CSV_SEPARATOR, ";");
-		properties.setProperty(ConfigFacadeEjb.TEMP_FILES_PATH, TMP_PATH);
+		resetProperties();
 
 		try {
 			Field instance = InfoProvider.class.getDeclaredField("instance");
@@ -104,6 +102,14 @@ public class MockProducer {
 			SORMAS_TO_SORMAS_REST_CLIENT,
 			managedScheduledExecutorService);
 		wireMocks();
+		resetProperties();
+	}
+
+	private static void resetProperties() {
+		properties.clear();
+		properties.setProperty(ConfigFacadeEjb.COUNTRY_NAME, "nigeria");
+		properties.setProperty(ConfigFacadeEjb.CSV_SEPARATOR, ";");
+		properties.setProperty(ConfigFacadeEjb.TEMP_FILES_PATH, TMP_PATH);
 	}
 
 	public static void wireMocks() {

